### PR TITLE
feat: add cart page and cart functionality

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
- Add Cart page.
- Add Cart functionality.
## Screenshots
Screenshot (221)
## Deploy Preview Link: https://deploy-preview-6--meck-store-app.netlify.app/

### CODE REVIEW

File: src/components/PriceCard/PriceCard.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { useCart } from "../../contexts";
+import "./PriceCard.css";
+
+const PriceCard = () => {
+  const { cartItems } = useCart();
+
+  const amountCalculator = (accu, current) =>
+    (accu += current.price * current.qty);
```
Comment (ivinayakg) [inline src/components/PriceCard/PriceCard.js:8]: there is no need to assign in the  "accu"

File: src/pages/Cart/Cart.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      const response = await axios.delete(`/api/user/cart/${productId}`, {
+        headers: { authorization: encodedToken },
+      });
+      setCartItems(response.data.cart);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const productsQuantityHandler = async (productId, btnType) => {
```
Comment (ivinayakg) [inline src/pages/Cart/Cart.js:24]: you can put these functions in a  utils folder file like cartUtils.js

File: src/pages/Login/Login.js [hunk 3]
```diff
@@ -28,6 +28,26 @@ const Login = () => {
     }
   };
 
+  const guestLoginHandler = async (e) => {
```
Comment (ivinayakg) [inline src/pages/Login/Login.js:31]: really nice feature.

**Review by ivinayakg — APPROVED** [review]
Your code is really clean hard to find any mistake, therefore found null.
Great Work 👍

**Review by dusariudaykumar — APPROVED** [review]
- Code looks clean.
- The directory structure looks clean and clear

File: src/components/PriceCard/PriceCard.js (hunk 1 above)
Comment (devdeadviz) [reply src/components/PriceCard/PriceCard.js:8]: Okay

File: src/pages/Login/Login.js (hunk 3 above)
Comment (devdeadviz) [reply src/pages/Login/Login.js:31]: Thank you

File: src/pages/Cart/Cart.js (hunk 2 above)
Comment (devdeadviz) [reply src/pages/Cart/Cart.js:24]: Yeah! I have planned to do refactoring at the end of the project building.

**Pull request merged**